### DISCUSSION TITLE
Steering wheel controls

### DISCUSSION THREAD

**Discussion by kentherzz** [ludwig-v/wireless-carplay-dongle-reverse-engineering#47] [General]
I have a Golf MK7 GTE 2016, with MIB2 headunit. Everything works fine except for fast-forwarding with steering wheel buttons - I can only skip tracks but I cannot seek by holding the button. Anyone else have same issue with Golfs, or any other car? :)

This works perfectly fine with wired CarPlay on the same car.

**Comment by milesking** [comment]
Civic 10th got the same issue. Stock wired CarPlay supports ff by long-pressing the steering wheel arrow button.

**Reply by kentherzz** [reply]
Thanks for the answer. Is the problem consistent on any firmware versions you have tried? I am new to the dongle and I have just tried the latest 2 firmware versions without any luck. I am wondering it worked at all on older firmware versions...

**Comment by spiegelt** [comment]
i have the same issue with an Opel Insignia B/ Multimedia Navi Pro. With Autokit 2021 03 06 :-(

**Comment by paskovitch** [comment]
I have same issue Golf MK7 2017 at latest Autokit firmware.

**Comment by Blade44git** [comment]
Same in Audi mmi, fast forward not work on long press skip button

**Comment by PhoneixDownn** [comment]
Any update on this? Mr-fix on YouTube writes in a spreadsheet that this works now with CPlay2Air adapter but not Carlinkit 3.0. Isn’t these dongles the same stuff where the only difference being the firmware?
 
Source:
https://docs.google.com/spreadsheets/d/1EMCOfEgc6PSrUTRv7o8LPsh3WMs2J6uXPPXKCg-_vHo/htmlview

I hope someone can test fast forwarding (with the steering wheel) again in their car and report back:-)

**Comment by spiegelt** [comment]
is anything in development?
